server/channels/app/post_metadata.go

```diff
@@ -169,13 +171,17 @@ func (a *App) getEmbedsAndImages(c request.CTX, post *model.Post, isNewPost bool
 		post.Metadata = &model.PostMetadata{}
 	}
 
-	// Embeds and image dimensions
-	firstLink, images := a.getFirstLinkAndImages(post.Message)
-
 	if post.Metadata.Embeds == nil {
 		post.Metadata.Embeds = []*model.PostEmbed{}
 	}
 
+	if unsafeLinksProp := post.GetProp(UnsafeLinksPostProp); unsafeLinksProp != nil && unsafeLinksProp.(string) == "true" {
```

Comment:
I think we might need to also write `value, ok := unsafeLinksProp.(string); ok && value == "true"` to avoid this panicking if that prop is not a string